app/views/paginable/orgs/_index.html.erb

```diff
@@ -5,7 +5,7 @@
         <th><%= _('Organisation') %>&nbsp;<%= paginable_sort_link('orgs.name') %></th>
         <th><%= _('Administrator Email') %>&nbsp;<%= paginable_sort_link('orgs.contact_email') %></th>
         <th><%= _('Organisation Type(s)') %>&nbsp;<%= paginable_sort_link('orgs.org_type') %></th>
-        <th><%= _('Templates') %>&nbsp;<%= paginable_sort_link('orgs.updated_at') %></th>
+        <th><%= _('Templates') %></th>
```

Comment:
good you removed this sortable field. Sorting by updated_at when the field name was Templates seems odd